pkg/webhooks/user/user.go

```diff
@@ -110,21 +175,47 @@ func (s *UserWebhook) authorized(request admissionctl.Request) admissionctl.Resp
 		return ret
 	}
 
-	// If it's a protected user kind, perform other checks to require requestors be a member of an admin group.
-	if strings.HasSuffix(userReq.Metadata.Name, protectedUserSuffix) {
-		for _, userGroup := range request.AdmissionRequest.UserInfo.Groups {
-			if utils.SliceContains(userGroup, adminGroups) {
-				ret = admissionctl.Allowed("Members of admin group are allowed")
+	// Red Hat associates follow special rules about who can make changes and who
+	// can have accounts
+	if s.isRedHatAssociate(userReq) {
+		// Load protected users from Kubernetes
+		if err = s.loadUsers(); err != nil {
+			fmt.Printf("Error loading users +%v\n", err.Error())
+			log.Error(err, "Couldn't load any valid users! No @redhat.com associates may have an account!")
```

Comment:
Would it actually error if there are no @redhat.com users? As we look at backplane replacing our in-cluster access we _won't_ (eventually) have users in cluster.  We shouldn't see this section hit in that case so it's ok, right?

Reply:
What will happen here is if `s.loadUsers()` returns an error we consider that to be some kind of internal problem (because there are always users) and return that error condition to the requestor.